lib/bolt/executor.rb

```diff
@@ -117,6 +118,11 @@ def with_node_logging(description, batch)
     end
     private :with_node_logging
 
+    def update_options(options)
```

Comment:
`update_run_as` might be more descriptive unless we plan to add other options to this in the future

Reply:
Ah yeah this is leftover from the original implementation when it was injecting the plan_context